### ISSUE TITLE
simplified reporting option

### ISSUE THREAD

**Issue by jvolkening** [marcelm/cutadapt#112]
Hello,

Lately I have been using cutadapt as part of batch scripts processing dozens of FASTQ files at a time. I wanted a simpler output from cutadapt for the sake of more easily checking the pre-processing steps. Originally I was going to parse the default verbose report format from cutadapt, but decided it would be just as quick and more useful to patch cutadapt directly to add such an option. I have posted a patch here:

https://gist.github.com/jvolkening/c34e1258dc8ac905dba5

It adds a "--succinct" option which tells cutadapt to output a single, tab-delimited report with a minimum of relevant fields so that, for instance, multiple sequential calls to cutadapt will produce a tab-delimited table of report data rather than a long text file. Feel free to either ignore or incorporate as you see fit. I should note that Python is not my first language, and that the patch has not been extensively tested. I will report back if I find any problems with further regular usage.

Regards,
Jeremy

**Comment by marcelm** [comment]
Nice! I have been thinking about a simplified report for a while, but never got around to implementing it. I’ll probably have to adjust the patch a bit, though. And I’m also not sure when I have time for this.

**Comment by marcelm** [comment]
Hi, I realize it’s been a long time and this may not be interesting anymore to to you, but perhaps you’ll still appreciate to know that I’ve implemented a "minimal reporting" option based on your patch. I have kept the order of fields as you implemented it, but there two other differences: First, the option is called `--report=minimal`, which will make it easier to add other report formats in the future. Also, to make it more useful without having to constantly refer to the manual, I added a header line as well (so it’s two-line output). It’s then not quite as easy to concatenate the individual reports into one longer one, but `cutadapt ... | tail -n1` is simple enough, so I think it’s worth it.

Thanks again!
